docs/architecture/modernize-desktop/example-migration-to-net-core.md

```diff
@@ -0,0 +1,365 @@
+![.csproj file ](media/example-migration-to-net-core/csproj-file.png)
+
+As mentioned before, .NET Core project has a more compact style and we need to
+migrate the project structure to the new .NET Core SDK style.
+
+Select the Windows Forms project in the solution explorer and do *right click
+-> Unload Project -> Edit*
+
+![Unload Project](media/example-migration-to-net-core/unload-project.png)
+
+Now we can update the .csproj file. We will delete all the content and replace
+it with:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+    <PropertyGroup>
+        <OutputType>WinExe</OutputType>
+        <TargetFramework>netcoreapp3.0</TargetFramework>
+        <UseWindowsForms>true</UseWindowsForms>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    </PropertyGroup>
+</Project>
+```
+
+If we save and reload the project, we are done updating the project file. Now
+the project is targeting the .NET Core.
+
+If we compile the project at this point, we find some errors related to the WCF
+client reference. Since this is autogenerated code, we must regenerate it to
+target .NET Core.
+
+![Errors](media/example-migration-to-net-core/errors.png)
+
+We can delete the `Reference.cs` file and generate a new Service Client.
+
+Right-click over *Connected Services* and select the "*Add Connected Service*"
+option.
+
+![Add Connected Service](media/example-migration-to-net-core/add-connected-service.png)
+
+This will open the Connected Services window where we select the Microsoft WCF
+Web Service option.
+
+![Connected Services window](media/example-migration-to-net-core/connected-services-window.png)
+
+If we have the WCF Service in the same solution as we have, we can choose to
+select the Discover option instead of specifying a service URL.
+
+![Configure WCF Web Service Reference window](media/example-migration-to-net-core/configure-wcf-reference.png)
+
+Once the service is located, the tool reflects the API contract implemented by
+the service. We change the name of the Namespace to be eShopServiceReference:
+
+![API contract](media/example-migration-to-net-core/api-contract.png)
+
+We click on *Finish* and after a while, we will see the generated code.
+
+We can see three generated documents:
+
+1. Getting Started: just a link to GitHub to provide some info on WCF
+2. `ConnectedService.json`: configuration parameters to connect to the service.
+3. `Reference.cs`: this is the actual WCF client code.
+
+![Generated documents](media/example-migration-to-net-core/generated-documents.png)
+
+If we compile again, we see many errors coming from `.cs` files inside the Helper
+folder. This folder was present in the .NET Framework version but not included
+in the old .csproj. But with the new SDK project style every code file present
+underneath the project file location is included by default, meaning that the
+new .NET Core project tries to compile the files inside the Helper folder. Since
+it is not needed, we can safely delete it.
```

Comment:
Here I'm assuming that it in the last sentence refers to the folder. Please review.

```suggestion
If you compile again, you'll see many errors coming from *.cs* files inside the *Helper*
folder. This folder was present in the .NET Framework version but not included
in the old .csproj. But with the new SDK-style project, every code file present
underneath the project file location is included by default. That is, the
new .NET Core project tries to compile the files inside the *Helper* folder. Since
that folder isn't needed, you can safely delete it.
```